tests/test_channel_metrics/test_completely_bounded_trace_norm.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+from toqito.channels import dephasing
+def test_diamond_norm_quantum_channel():
+    """The diamond norm of a quantum channel is 1"""
+    phi = dephasing(1)
+    np.testing.assert_equal(np.isclose(diamond_norm(phi), 1, atol=1e-3), True)
```

Comment:
Since this function will return `1` in the event that the channel is quantum, I wonder if we need the `isclose` statement.